content/guides/userguides/terra-export.md

```diff
@@ -0,0 +1,126 @@
+---
+path: "/guides/userguides/consumer-vignettes/export-to-terra"
+date: "2019-09-17"
+title: "Exporting HCA Data to Terra"
+subTitle: ""
+---
+
+Exporting search results from the HCA Data Explorer to Terra
+============================================================
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
```

Comment:
Many / most  users will not be familiar with Terra as it is a new tool. I think we might add see a brief overview of what terra is and what dockstore is for folks that do not know with links out to terra and dockstore to learn more.

Reply:
Added two sections (Overview of Terra and Overview of Dockstore) that link to relevant pages/documentation